### PR TITLE
add SimpleX Chat

### PR DESCRIPTION
**Pull request by epoberezkin**

### CODE REVIEW

**Comment by tbaka** [comment]
howdy, I wanted to let you know that we're starting the review process for your submission. We have a thirty day window for new submissions to release _after_ they are passing tests. This would put you on October 16th.

File: apps/linode-marketplace-simplex/logo.svg
Comment (tbaka) [inline apps/linode-marketplace-simplex/logo.svg:1]: The logo svgs need a full name attached. will you rename this to `/apps/linode-marketplace-simplex/assets/simplexchat.svg`

File: apps/linode-marketplace-simplex/requirements.txt
Comment (tbaka) [inline apps/linode-marketplace-simplex/requirements.txt:1]: after EOF please add the following block. 
`### testing reqs ###
#molecule==5.1.0
#molecule-vagrant==2.0.0
#python-vagrant==0.5.15`

**Review by tbaka — CHANGES_REQUESTED** [review]
This passes deploy, and we can merge to develop when requested changes are made

**Comment by epoberezkin** [comment]
hello! cc @shumvgolove

File: apps/linode-marketplace-simplex/logo.svg
Comment (shumvgolove) [reply apps/linode-marketplace-simplex/logo.svg:1]: Done!

File: apps/linode-marketplace-simplex/requirements.txt
Comment (shumvgolove) [reply apps/linode-marketplace-simplex/requirements.txt:1]: Done!

**Comment by shumvgolove** [comment]
👋 @tbaka Could you take a look?

**Review by tbaka — APPROVED** [review]
Looks good!

**Comment by tbaka** [comment]
We have your app scheduled for release on October 16th. Let me know if you have any questions.

**Comment by epoberezkin** [comment]
@tbaka thank you!

**Pull request merged**